templates/libraries/okhttp-gson/pojo.mustache

```diff
@@ -599,23 +597,4 @@ public class {{classname}} {{#parent}}extends {{{.}}} {{/parent}}{{#vendorExtens
   }
 {{/hasChildren}}
 
- /**
```

Comment:
Why not include these methods? Might be of use for some devs

Reply:
Because they depend on this other object `JSON` (`import {{invokerPackage}}.JSON`) which is not a model and only works for a single service at a time. We can investigate how to integrate it to the library on another ticket.